projects/schematics/src/shared/utils/lib-utils_spec.ts

```diff
@@ -130,149 +128,80 @@ describe('Lib utils', () => {
         .toPromise();
     });
 
-    it('should add i18n', async () => {
-      const rule = addLibraryFeature(
-        appModulePath,
-        BASE_OPTIONS,
-        BASE_FEATURE_CONFIG
-      );
-      const result = await schematicRunner.callRule(rule, appTree).toPromise();
+    it('should create new feature module', () => {});
```

Comment:
Are we planning to implement these in this PR?

Reply:
Nope

Reply:
this is just a reminder when implementing rest of the test